test/unittest/rtps/resources/timedevent/TimedEventTests.cpp

```diff
@@ -673,16 +450,6 @@ TEST(TimedEventMultithread, EventNonAutoDestruc_FourAutoRestart)
 
     // Wait a prudential time before delete the event.
     std::this_thread::sleep_for(std::chrono::milliseconds(200));
```

Comment:
What are the expectations in this test?

Reply:
Tests an event can be used from several thread without crashing.

Reply:
:+1: I thought it would be something like that